build.gradle

```diff
@@ -52,6 +52,10 @@ allprojects {
     tasks.withType(Javadoc) {
         options.addBooleanOption('Xdoclint:all,-missing', true)
     }
+
+    tasks.withType(JavaCompile) {
+      options.compilerArgs << "-Xlint:unchecked" << "-Xlint:deprecation"
```

Comment:
temporary?

Reply:
No. We want to add these. I'm surprised they were not here before.